package.json

```
@@ -6,15 +6,19 @@
     "autoprefix": "postcss --use autoprefixer -o network-api/networkapi/frontend/_css/main.compiled.css network-api/networkapi/frontend/_css/main.compiled.css",
     "build:common": "shx rm -rf network-api/networkapi/frontend && shx mkdir -p network-api/networkapi/frontend/_js && run-p build:images build:sass && npm run build:contribute",
     "build:contribute": "shx cp contribute.json network-api/networkapi/frontend/",
+    "build:dev": "npm run build:common && npm run build:js:dev",
```

`npm-run-all` is already installed, so:
```suggestion
    "build:dev": "run-s build:common build:js:dev",
```